Client/mods/deathmatch/logic/CDownloadableResource.cpp

```diff
@@ -10,23 +10,20 @@
  *****************************************************************************/
 
 #include <StdInc.h>
+#include <future>     // return value from SharedUtil::async
+
+#define INVALIDATE_OLD_FUTURE if (m_GenerateChecksumFuture.valid()) m_GenerateChecksumFuture = std::future<CChecksum>();
```

Comment:
what's wrong with a private function?

Reply:
I felt like this is neater, than declaring a 4-5 line function.

Reply:
macros are undesirable

Reply:
> macros are undesirable
agreed

Reply:
agreed